CONTRIBUTING.rst

```diff
@@ -0,0 +1,263 @@
+
+Testing
+~~~~~~~
+
+Please add tests, especially with new code. As of now, we have unit tests (in `/test`). They are straightforward to augment, but we understand
+it's the extra mile; it would still be appreciated if you provide something lighter (eg. in the commit messages or in the PR or issue text)
+that demonstrates that an issue was fixed, or a feature is functional.
+
+Consider that if you add test cases, they will ensure that your feature still works in the future.
+
+Licensing
+~~~~~~~~~
+
+Ensure that you are the original author of your changes, and if that is not the case, ensure that the borrowed/adapted code is compatible with
+the `project's license`. --> tODO
+
+
+Committing
+----------
+
+Commit Titles
+~~~~~~~~~~~~~
+
+Provide a concise and self-descriptive title (avoid > 80 characters).
+You may "scope" the title using the applicable command name(s), folder
+or other "module" as a prefix. If a commit is responsible for fixing
+an issue, post-fix the description with ``(fixes #ISSUE_NUMBER)``.
+
+Examples:
+
+::
+
+    testing: add testing function for validation metrics
+    loader: add timer
+    documentation: add slice_axis to the config files
+
+Commit Sequences
+~~~~~~~~~~~~~~~~
+
+Update your branch to be baseline on the latest master if new
+developments were merged while you were developing. Please prefer
+**rebasing** to merging, as explained in `this
+tutorial <https://coderwall.com/p/7aymfa/please-oh-please-use-git-pull-rebase>`__.
+Note that if you do rebases after review have started, they will be
+cancelled, so at this point it may be more appropriate to do a pull.
+
+Clean-up your commit sequence. If your are not familiar with git, `this
+good tutorial <https://www.atlassian.com/git/tutorials/rewriting-history>`__
+on the subject may help you.
+
+Focus on committing 1 logical change at a time. See `this article
+<https://github.com/erlang/otp/wiki/writing-good-commit-messages>`__
+on the subject.
+
+
+Submitting a Pull Request
+-------------------------
+
+PR Title
+~~~~~~~~
+
+For each new release, so please follow the following rules:
+
+-  Provide a concise and self-descriptive title (see 'Issue Title').
+-  Do not include the applicable issue number in the title, do it in the PR body (see 'PR Body' section).
+-  If the PR is not ready for review, convert it to a draft.
+
+PR Body
+~~~~~~~
+
+Describe what the PR is about, explain the approach and possible
+drawbacks. Don't hesitate to repeat some of the text from the related
+issue (easier to read than having to click on the link).
+
+If the PR fixes issue(s), indicate it after your introduction:
+``Fixes #XXXX, Fixes #YYYY``. Note: it is important to respect the
+syntax above so that the issue(s) will be closed upon merging the PR.
+
+PR Labels
+~~~~~~~~~
+
+Add the labels that best describe your PR ('enhancement', 'documentation', etc...
```

Comment:
do it as done [here](https://github.com/neuropoly/spinalcordtoolbox/blob/master/CONTRIBUTING.rst#pr-labels).

which also means: please create appropriate labels categories